packages/components/toaster/src/Toaster.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { CloseButton, ProgressBar } from './customOptions';
+import { getToasterTypesStyles } from './utils';
+import { useToaster } from './hooks';
+import type { ToasterProps } from './toaster.types';
+
+export const Toaster = ({
+  id,
+  typeToaster = 'success',
+  position,
+  isInStack = false,
+  duration,
```

Comment:
What if we set a default duration? 5000ms for example?

Reply:
I left it blank intentionally to allow users to choose a duration that best fits their needs. However, setted a default duration could be a helpful starting point while still allowing for customization. I would also like to know what is the default setting for this kind of props, is better ever set a default value or leave it blank to the user choose the default value?

Reply:
Set a default value in number type.